secure/authenticating-pipelines-repos-using-secrets.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="authenticating-pipelines-repos-using-secrets"]
+= Authenticating pipelines with repositories using secrets
+:context: authenticating-pipelines-repos-using-secrets
+
+toc::[]
+
+Pipelines and tasks can require credentials to authenticate with Git repositories and container repositories. In {pipelines-title}, you can use secrets to authenticate pipeline runs and task runs that interact with a Git repository or container repository during execution.
+
+A secret for authentication with a Git repository is known as a _Git secret_.
+
+A pipeline run or a task run gains access to the secrets through an associated service account. Alternatively, you can define a workspace in the pipeline or task and bind the secret to the workspace.
+
+
+[id="providing-secrets-using-service-accounts_{context}"]
+== Providing secrets using service accounts
+
+You can use service accounts to provide secrets for authentication with Git repositories and container repositories.
+
+You can associate a secret with a service account. The information in the secret becomes available to the tasks that run under this service account.
+
```

Comment:
I think since all the procedural modules in this section use the oc CLI, it would be good to have a Prereqs section defining that as a prereq:
```suggestion

[id="providing-secrets-using-service-accounts-prereqs_{context}"]
=== Prerequisites
* You have installed the OpenShift CLI (`oc`)

```

Not honestly sure if the id/anchor is needed. Optionally, if you prefer then you can add this as a prereq in each procedural module that includes the use of `oc` instead of putting it here.

Reply:
Did this before the subdivisions.